src/foam/core/AbstractPropertyInfo.java

```diff
@@ -79,4 +90,9 @@ public void toXML(FObject obj, Document doc, Element objElement) {
       objElement.appendChild(prop);
     }
   }
+
+  @Override
+  public SQLType getSqlType() {
+    return SQLType.VARCHAR;
```

Comment:
We should include the 'width' for String properties, otherwise becomes very inefficient.

Reply:
Yeah good point. We would need some way to handle changes to the width property. Not sure how frequent that may be though.

Reply:
We created a separate DAO in the system to record the version #'s of the models for each table in the database. If it changed we would check for a "Migration Strategy" which would be applied to migrate the data. Typically we would just stream the data to a file, rebuilt the table and then reload, but you could have other tables. Some database give bad performance after you've 'alter'-ed them. But I don't think we need to worry about this for now.